jsnlog/jsnlog.d.ts

```diff
@@ -0,0 +1,116 @@
+// Type definitions for JSNLog v2.11.0+
+// Project: https://github.com/mperdeck/jsnlog.js
+// Definitions by: Mattijs Perdeck <https://github.com/mperdeck>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+// -------------------------------
+// Full documentation is at 
+// http://jsnlog.com
+// -------------------------------
+
+﻿/**
+* Copyright 2015 Mattijs Perdeck.
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE
this repository is under MIT license.